vehicle/polestar/types.go

```diff
@@ -3,9 +3,10 @@ package polestar
 import "time"
 
 type Token struct {
-	AccessToken  string `graphql:"access_token"`
-	RefreshToken string `graphql:"refresh_token"`
-	ExpiresIn    int    `graphql:"expires_in"`
+	AccessToken  string `json:"access_token"`
```

Comment:
…and this looks like your normal oauth2.Token?

Reply:
looks like 

```
{
  "scope": "openid profile email customer:attributes",
  "client_id": "l3oopkc_10",
  "grntid": "pgy8pMOexIftRPxn2d1zpzEtXKAvaTh5",
  "iss": "https://polestarid.eu.polestar.com",
  "iat": 1733862755,
  "jti": "j0WZYyVTPB58DMAsSkImGN",
  "market": "DE",
  "acr": "urn:polestarid:aal:bronze:any",
  "aud": "l3oopkc_10",
  "firstName": "Thomas",
  "lastName": "Leininger",
  "sub": "750d8bdf-cfdd-4b57-8863-545b963598f2",
  "mobile": "+49**********",
  "pi.sri": "yZ-bFpFvgeXMjGdKSLOmhHUoR58..WKVj.MJUGMYhCMP1AwOj644WNysHjm",
  "userName": "rostbeule@gmail.com",
  "email": "rostbeule@gmail.com",
  "exp": 1733863055
}
```

But facing 2 warnings:
Looks like your JWT header and signature is not encoded correctly using base64 blablabla...